components/ipfs/DEPS

```diff
@@ -1,11 +1,4 @@
 include_rules = [
-  "+content/public/browser",
-  "+content/public/common",
-  "+components/base32",
-  "+third_party/re2",
-  "+services/network/public",
-  "+storage/browser",
-  "+third_party/blink/public/mojom",
-  "+ui/base",
-  "+ui/native_theme",
-]
+  "+chrome/common",
```

Comment:
that's not ok to depend on chrome from components

Reply:
Follow up issue?